gm4_blast_furnaces/data/blast_furnaces/functions/pulse_check.mcfunction

```diff
@@ -0,0 +1,4 @@
+#compares assigned run tick to current tick and calls main
+execute if score current_tick gm4_clock_tick = blast_furnaces gm4_clock_tick run function blast_furnaces:main
+
+execute as @e[type=armor_stand,tag=gm4_bf_output,tag=gm4_bf_could_smelt] at @s if block ^ ^ ^1 furnace{CookTime:199s} run function blast_furnaces:smelt
```

Comment:
Instead of checking ever blast furnace entity each tick, you should create a fake player in the bf_data scoreboard that stores whether any blast furnaces are currently within 16 ticks of smelting an item. Checking a fake player every tick is far more efficient than checking every armorstand's tags each tick.

Reply:
Aren't we already doing this in [`check_cook_time.mcfunction`](gm4_blast_furnaces/data/blast_furnaces/functions/check_cook_time.mcfunction)?